types/minimatch/index.d.ts

```diff
@@ -1,33 +1,57 @@
-// Type definitions for Minimatch 3.0
+// Type definitions for minimatch 5.1
 // Project: https://github.com/isaacs/minimatch
 // Definitions by: vvakame <https://github.com/vvakame>
 //                 Shant Marouti <https://github.com/shantmarouti>
+//                 BendingBender <https://github.com/BendingBender>
```

Comment:
Just checking; are you looking to become an owner/reviewer or just looking for authorship credit? https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
Yes, that's what I wanted.